pkg/operator/ceph/object/rgw.go

```diff
@@ -273,8 +272,8 @@ func (c *clusterConfig) deleteStore() error {
 	// Delete rgw CephX keys
 	for i := 0; i < int(c.store.Spec.Gateway.Instances); i++ {
 		daemonLetterID := k8sutil.IndexToName(i)
-		keyName := fmt.Sprintf("client.%s.%s", strings.Replace(c.store.Name, "-", ".", -1), daemonLetterID)
-		err := cephclient.AuthDelete(c.context, c.store.Namespace, keyName)
+		depNameToRemove := fmt.Sprintf("%s-%s-%s", AppName, c.store.Name, daemonLetterID)
```

Comment:
Why naming this `depNameToRemove` if we remove a key?

Reply:
Because this doesn't represent the key to remove but more the rgw entity to remove (same logic naming than in the downscale part https://github.com/rook/rook/blob/eed14341ccc005e2373f50902a27946bdb1d5dd1/pkg/operator/ceph/object/rgw.go#L190)
But I don't have strong opinion here and can change the code to have this

> keyName := generateCephXUser(fmt.Sprintf("%s-%s-%s", AppName, c.store.Name, daemonLetterID))
		err := cephclient.AuthDelete(c.context, c.store.Namespace, keyName)

Let me know if you prefer this

Reply:
If it's the same logic as above already, sounds good, thanks!